deps/win/oibus_setup.iss

```diff
@@ -237,10 +238,10 @@ begin;
             Result := False
           else
           begin
-            FileContent := 'echo Stopping OIBus service...' + #13#10
-              + 'nssm.exe stop OIBus >nul 2>&1' + #13#10
+            FileContent := 'echo Stopping OIBus service... ' + #13#10
+              + 'nssm.exe stop OIBus >nul 2>&1 &> /dev/null' + #13#10
               + '@echo Starting OIBus in the console...' + #13#10
-              + '"' + ExpandConstant('{app}') + '\oibus.exe" --config "' + ConfigFilePath + '"'
+              + '"' + ExpandConstant('{app}') + '\oibus.exe" --config "' + ConfigFilePath + '" &> /dev/null'
```

Comment:
I told you to do that in a separate PR because it focus on something different that I don't want to test this feature in this PR. 
Moreover, from what I see, you do not offer the user the 'hideConsole' as an option, it seems always hidden. Lastly this seems to hide messages but the terminal itself seems to stay open which is not what the issue #1428  is asking.